pkg/controller/expcertificates/requestmanager/requestmanager_controller.go

```diff
@@ -310,15 +310,20 @@ func (c *controller) createNewCertificateRequest(ctx context.Context, crt *cmapi
 		return err
 	}
 
+	annotations := make(map[string]string, len(crt.Annotations)+3)
```

Comment:
A map with a fixed length, quite honestly the first time I saw this... Any particular reason for this?